mlflow/java/client/src/test/java/org/mlflow/tracking/MlflowClientTest.java

```diff
@@ -550,8 +550,10 @@ public void testUseArtifactRepository() throws IOException {
     String content = "Hello, Worldz!";
 
     File tempFile = Files.createTempFile(getClass().getSimpleName(), ".txt").toFile();
+    File tempDir = Files.createTempDirectory("tempDir").toFile();
     FileUtils.writeStringToFile(tempFile, content, StandardCharsets.UTF_8);
     client.logArtifact(runId, tempFile);
+    client.logArtifact(runId, tempDir);
```

Comment:
Nice, let's also verify that `tempDir` was logged & has the right contents (maybe we can add a file under it & check that it shows up)?